observabilityclient/v1/base.py

```diff
@@ -0,0 +1,123 @@
+#   Copyright 2022 Red Hat, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+
+import argparse
+import jinja2
+import os
+import yaml
+
+from osc_lib.command import command
+from osc_lib.i18n import _
+
+from observabilityclient.utils import runner
+from observabilityclient.utils import shell
+
+
+OBSLIBDIR = shell.file_check('/usr/share/osp-observability', 'directory')
+OBSWRKDIR = shell.file_check('/var/lib/osp-observability', 'directory')
```

Comment:
This ends up requiring execution as root by default - perhaps it should be a /tmp dir instead?

Reply:
Negative. osp-observability-ansible RPM will have to install it's contents to /usr/share folder and regarding of described purpose [1] of /var/lib I think it's the proper place for most the data saved there. Although you're right the temporary data should not have that directory as basedir. I'm going to rebuild my dev environment and make it work with the /var/lib and regular users and move the tmpdir basedir to /tmp/observability/. Make sense?

Reply:
Yes, it was really just the temporary data that I'm concerned about; though I see the current instructions contain `sudo chmod a+rwx /var/lib/osp-observability` which seems like there is still mishandling of temporary files (I'll check the code though, maybe the a+w was a leftover?)

EDIT: so it's not a leftover, I guess the idea is that any user could generate the files in /var/lib/osp-observability/ and they'd be re-used by future users. I'm not really familiar with the conventions around this in the openstack CLI tool. My experience has always been that artifacts are produced in the local directory by default, but if this is also acceptable, I have no serious problem with it.

Reply:
Meh I completely missed that I promised moving temp data out of /var/lib. Fixing it now.